public/usage-examples/graphics/current_clip/current-clip-1-simple-oop.cs

```diff
@@ -0,0 +1,24 @@
+using SplashKitSDK;
+
+public class Program
+{
+    public static void Main()
+    {   
+        // Open a window with the title "Current Clip" and dimensions
+        Window window= SplashKit.OpenWindow("Current Clip", 800, 600);
+        
+        // Push a clipping area
+        Rectangle rectangle = SplashKit.CurrentClip();
+        // Display information about the current clipping area (Width and Height) in the window
+        SplashKit.DrawText($"Current Clip: Width:{rectangle.Width} Height:{rectangle.Height}", Color.Black, "Arial", 24, 100, 100);
```

Comment:
Better use "writeline" here to print the result on the terminal, otherwise if the user creates a smaller window would not able to see the full  text.
![image](https://github.com/user-attachments/assets/9920eb44-4233-4751-81cb-169cac61a99e)